src/main/java/com/example/Student/Repositery/StudentRepositery.java

```diff
@@ -15,6 +18,15 @@ public interface StudentRepositery  extends JpaRepository<Student, Integer>{
     public List<Student> findAllByDepartment(String department);
 	
 	public List<Student> findByRollnoBetween(int a ,int b);
+	
+	
+	@Query("select s from Student s WHERE s.department=:d")
+	public List<Student> getAllByDepartment( @Param ("d") String department);
+	
+	//update email and phone number
+	@Modifying
+	@Query("update Student s set s.email =:e,s.rollno=:r where s.id=:i")
```

Comment:
Does the column exist by default in the table?

Reply:
No by default. these is a part of Student entity.I just updated these details for a specific Student .

Reply:
ok let me check your entity class